src/interfaces/attribute-decorator-options.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { PropertyConverter } from './property-converter.interface';
```

Comment:
I am not sure if it´s necessary to put interfaces in an interface directory, since they also contain interface in the name.

Reply:
In that case I would rather create a folder for each decorator.
ie. `/src/decorators/attribute` folder will contain `attribute.decorator.ts` and `attribute-decorator-options.interface.ts` files.

What do you think?